web-console/knikubevirt/Create-vm/step-2-networking/create-vm-networking.md

```diff
@@ -11,6 +11,10 @@ Clicking on 'Add NIC' will pop up a "Create NIC" modal. The NIC name is prepopul
 ![Type - Masquerade](img/3-02.png)
 In case the user selects Masquerade on the Type field, new indented fields will appear.
 
+When adding a new NIC we allow the user to add one port with multiple bindings, so once the user enters the name for the first port we have another field popping up below where they can add port bindings.
```

Comment:
I think, this does not translate well into the Interface definitions (I don't understand the meaning of binding here).

Definition of the port:

##### Port
- name (unique string)
- port (number 0 < x < 65536)
- protocol (TCP, UDP)

We should be able to define a list of Ports (name, port, protocol) which will be then forwarded and exposed to the virtual machine when `masquerade` is selected.

Ways to implement it?
- Another dialog is not feasible because of location in VM  wizard
- inline edit list?

we also need validation errors for the name

Reply:
Thanks, @suomiy for explaining this. Please review.